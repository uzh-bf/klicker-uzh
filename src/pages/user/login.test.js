/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Login from './login'

describe('Snapshot-Testing', () => {
  it.skip('Works', () => {
    const component = renderer.create(<Login />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
