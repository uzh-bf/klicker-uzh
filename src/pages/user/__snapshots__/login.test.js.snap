// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot-Testing Works 1`] = `
<div
  className="jsx-2966695483 jsx-3979270099 commonLayout"
>
  <div
    className="jsx-212519681 staticLayout"
  >
    <main
      className="jsx-212519681 content"
    >
      <div
        className="jsx-2877924394 login"
      >
        <h1
          className="jsx-2877924394"
        >
          <span>
            Login
          </span>
        </h1>
        <div
          className="jsx-1235538274 formWithLinks"
        >
          <form
            action={undefined}
            className="ui error form"
            onSubmit={[Function]}
          >
            <div
              className="required field"
            >
              <label
                className="jsx-3360603769"
                forHtml="email"
              >
                Email
              </label>
              <div
                className="ui icon input"
                intl={
                  Object {
                    "defaultFormats": Object {},
                    "defaultLocale": "en",
                    "formatDate": [Function],
                    "formatHTMLMessage": [Function],
                    "formatMessage": [Function],
                    "formatNumber": [Function],
                    "formatPlural": [Function],
                    "formatRelative": [Function],
                    "formatTime": [Function],
                    "formats": Object {},
                    "formatters": Object {
                      "getDateTimeFormat": [Function],
                      "getMessageFormat": [Function],
                      "getNumberFormat": [Function],
                      "getPluralFormat": [Function],
                      "getRelativeFormat": [Function],
                    },
                    "locale": "en",
                    "messages": Object {},
                    "now": [Function],
                    "textComponent": "span",
                  }
                }
              >
                <input
                  autoFocus={true}
                  disabled={undefined}
                  name="email"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onDragStart={[Function]}
                  onDrop={[Function]}
                  onFocus={[Function]}
                  placeholder="Email"
                  tabIndex={undefined}
                  type="email"
                  value=""
                />
                <i
                  aria-hidden="true"
                  className="mail icon"
                />
              </div>
            </div>
            <div
              className="required field"
            >
              <label
                className="jsx-3360603769"
                forHtml="password"
              >
                Password
              </label>
              <div
                className="ui icon input"
                intl={
                  Object {
                    "defaultFormats": Object {},
                    "defaultLocale": "en",
                    "formatDate": [Function],
                    "formatHTMLMessage": [Function],
                    "formatMessage": [Function],
                    "formatNumber": [Function],
                    "formatPlural": [Function],
                    "formatRelative": [Function],
                    "formatTime": [Function],
                    "formats": Object {},
                    "formatters": Object {
                      "getDateTimeFormat": [Function],
                      "getMessageFormat": [Function],
                      "getNumberFormat": [Function],
                      "getPluralFormat": [Function],
                      "getRelativeFormat": [Function],
                    },
                    "locale": "en",
                    "messages": Object {},
                    "now": [Function],
                    "textComponent": "span",
                  }
                }
              >
                <input
                  disabled={undefined}
                  name="password"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onDragStart={[Function]}
                  onDrop={[Function]}
                  onFocus={[Function]}
                  placeholder="Password"
                  tabIndex={undefined}
                  type="password"
                  value=""
                />
                <i
                  aria-hidden="true"
                  className="privacy icon"
                />
              </div>
            </div>
            <div
              className="jsx-1235538274 actionArea"
            >
              <button
                className="ui primary disabled button semanticButton"
                disabled={true}
                onClick={[Function]}
                role="button"
                tabIndex={-1}
                type="submit"
              >
                Submit
              </button>
              <div
                className="jsx-1235538274 links"
              >
                <div
                  className="ui list"
                  role="list"
                >
                  <div
                    className="item"
                    data-value={undefined}
                    onClick={[Function]}
                    role="listitem"
                  >
                    <a
                      className="jsx-1235538274"
                      href="/user/requestPassword"
                      onClick={[Function]}
                    >
                      Forgot password?
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </form>
        </div>
      </div>
    </main>
    <footer
      className="jsx-212519681"
    >
      <hr
        className="jsx-212519681"
      />
      <p
        className="jsx-212519681"
      >
        Â©2017 IBF Teaching Center, Department of Banking and Finance, University of Zurich. All rights reserved.
        <br
          className="jsx-212519681"
        />
        Products and Services displayed herein are trademarks or registered trademarks of their respective owners.
      </p>
    </footer>
  </div>
</div>
`;
